src/olcs/MVTImageryProvider.js

```diff
@@ -29,14 +30,16 @@ export default class MVTImageryProvider {
     this.errorEvent = new Cesium.Event();
     this.credit = options.credit;
     this.hasAlphaChannel = true;
-    this.cache_ = {};
     this.styleFunction_ = options.styleFunction || (() => styles);
     this.projection_ = getProjection('EPSG:3857');
     this.emptyCanvas_ = document.createElement('canvas');
     this.emptyCanvas_.width = 1;
     this.emptyCanvas_.height = 1;
-    this.featuresCache = options.featuresCache || {};
     this.tileRectangle_ = new Cesium.Rectangle();
+    const cacheSize = options.cacheSize ? options.cacheSize : 50;
```

Comment:
Could be useful if we want to test with cacheSize of 0
```suggestion
    const cacheSize = options.cacheSize !== undefined ? options.cacheSize : 50;
```